include/dem/particle_wall_dmt_force.h

```diff
@@ -0,0 +1,220 @@
+/* ---------------------------------------------------------------------
+ * Copyright (C) 2019 - 2019 by the Lethe authors
```

Comment:
```suggestion
 * Copyright (C) 2019 - 2024 by the Lethe authors
```

We will try to change this soon but let's still change it when we can.